### PR TITLE
[samples] Ported helloworld from dart-tutorial-samples

### PR DESCRIPTION
**Pull request by jddeep**
[samples] Ported helloworld from dart-tutorial-samples

Fixes https://github.com/dart-lang/dart-tutorials-samples/issues/55

### CODE REVIEW

**Comment by jddeep** [comment]
@kwalrath Done. Have a look. Thanks :)

File: examples/helloworld/bin/helloworld.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+void main() {
+  print("Hello, World!");
```
Comment (kwalrath) [inline examples/helloworld/bin/helloworld.dart]: I just realized that this should be in single quotes, to match our style.

File: examples/helloworld/.gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Files and directories created by pub
```
Comment (kwalrath) [inline examples/helloworld/.gitignore]: I don't see `.gitignore` in other directories under `examples/`. We can remove this, right @johnpryan?

Comment (jddeep) [reply examples/helloworld/.gitignore]: Yup I realized it as well. There is already a parent `.gitignore`

**Comment by jddeep** [comment]
@kwalrath Updated. Have a look :)

File: src/_tutorials/server/cmdline.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Let's run a small program.
 
  1. Create a file called `helloworld.dart` that contains this code:
 
-    <?code-excerpt "misc/test/samples_test.dart (hello-world)"?>
```
Comment (kwalrath) [inline src/_tutorials/server/cmdline.md]: Oh wait a minute... I didn't realize that this code snippet was already tested and in the build.

**Pull request closed without merging**